pkg/bootstrap/types.go

```diff
@@ -33,7 +33,7 @@ type KuryrBootstrapResult struct {
 type OVNConfigBoostrapResult struct {
 	GatewayMode            string
 	EnableEgressIP         bool
-	DisableSNATMutlipleGWs bool
```

Comment:
wow! I had to stare at it for a good 5 seconds before realizing what the error was. Nice catch, at least they were consistently wrong and hence worked fine (I had spent sometime this week testing this out and this never caught my eye!) nice catch Dan.